.github/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ### New features since last release
 
+* `lightning.qubit` supports mid-circuit measurements.
```

Comment:
Actually that's not Release 0.35 any more. Should we create a new Release 0.36 section right now?

Reply:
No, it is generally done in a separate PR #626 .